### PR TITLE
[astronomer/issues#360] Add Velero to Cloud System Components

### PR DESCRIPTION
**Pull request by kaxil**
Issue: https://github.com/astronomer/issues/issues/360

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # Networks, Database, Kubernetes cluster, etc.
 module "gcp" {
   source              = "astronomer/astronomer-gcp/google"
+//  source              = "../terraform-google-astronomer-gcp"
   version             = "1.0.111"
```
Comment (kaxil) [inline main.tf]: @schnie @sjmiller609 The version needs to be updated after merging https://github.com/astronomer/terraform-google-astronomer-gcp/pull/29

File: main.tf [hunk 2]
```diff
@@ -34,7 +35,7 @@ module "system_components" {
   dependencies = [module.gcp.depended_on]
   source       = "astronomer/astronomer-system-components/kubernetes"
   version      = "0.1.7"
```
Comment (kaxil) [inline main.tf]: Same here. The version needs to be updated after merging https://github.com/astronomer/terraform-kubernetes-astronomer-system-components/pull/4

Comment (sjmiller609) [reply main.tf]: patched with new version

File: main.tf (hunk 1 above)
Comment (sjmiller609) [reply main.tf]: waiting for pipeline. this ought to be automatically added shortly

Comment (sjmiller609) [reply main.tf]: https://cloud.drone.io/astronomer/terraform-google-astronomer-gcp/121/1/3

**Pull request merged**